finance.reckoner.Reckoner.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: finance.reckoner.Reckoner
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: reckoner
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin ${FLATPAK_DEST}/reckoner ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/ ${FLATPAK_DEST}/share/applications/ ${FLATPAK_DEST}/share/appdata/
+      - cp reckoner-git/linux/flatpak/${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/
+      - cp reckoner-git/linux/flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/
+      - cp reckoner-git/linux/flatpak/${FLATPAK_ID}.appdata.xml ${FLATPAK_DEST}/share/appdata/
+      - cp -R bundle/* ${FLATPAK_DEST}/reckoner/
+      - ln -s ${FLATPAK_DEST}/reckoner/reckoner ${FLATPAK_DEST}/bin/reckoner
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        dest: bundle
+        url: https://codeberg.org/Reckoner/app/releases/download/v1.0.2/reckoner-1.0.2-x86_64-linux.tar.gz
+        sha256: 0ac1c115ad708c56f8f7815f214a09c201a08a783686100c706e40898dd62c6b
+      - type: git
+        url: https://codeberg.org/Reckoner/app.git
+        dest: reckoner-git
+        branch: flathub
```

Comment:
oh and none of these files are flatpak specific

Reply:
Not sure what you mean by that last comment: "none of these are flatpak specific".

I will create a new release tag when I have all the other issues resolved.

Reply:
`reckoner-git/linux/flatpak`  is where you have put these files that are NOT flatpak specific. The .desktop is a 20+ year old spec, with the icons. The AppStream spec isn't part of Flatpak, it's just that Flatpak mandates it because it does what is needed. While it doesn't matter here for the result, it's just blatantly wrong. Regular distro packages will also use them.

Reply:
> NEVER use a git branch. You need a commit or a tag.

Just to clarify, this is not an option. The build will fail.

Reply:
Ok, only one issue I need to figure out before pointing this to a code release bundle. 

The file picker for my Flutter dependency calls zenity and just errors right now. Is there anything I can add/change for the flatpak environment to forward a zenity call?

https://github.com/flathub/flathub/pull/4847#issuecomment-1891018237

Reply:
You can bundle zenity with the app. Use the GitHub search in Flathub org to find existing examples.